packages/base/card-api.gts

```diff
@@ -1297,7 +1297,10 @@ class LinksToMany<FieldT extends CardDefConstructor>
           cachedInstance[isSavedInstance] = true;
           return cachedInstance;
         }
-        let resourceId = new URL(value.links.self, relativeTo).href;
+        let resourceId = new URL(
+          value.links.self,
+          !Array.isArray(doc.data) ? doc.data.id : relativeTo,
+        ).href;
```

Comment:
Before change 

```
relativeTo.href = 'http://localhost:4201/experiments/sprint-task'
value.links.self = '../Tag/f00a2177-df10-4ad7-8eac-2fbf8448de98'
resourceId =  'http://localhost:4201/Tag/f00a2177-df10-4ad7-8eac-2fbf8448de98'
```

After change 

```
doc.data.id = 'http://localhost:4201/experiments/SprintTask/259a2df6-6f0e-4795-abcf-2d16c437fb97'
value.links.self = '../Tag/f00a2177-df10-4ad7-8eac-2fbf8448de98'
resourceId = 'http://localhost:4201/experiments/Tag/f00a2177-df10-4ad7-8eac-2fbf8448de98'
```

The resourceId should be relative to instance, not the module